test/test_images/recordevents/main.go

```diff
@@ -145,10 +153,13 @@ func main() {
 	if err := tracing.SetupStaticPublishing(logger.Sugar(), "", tracing.AlwaysSample); err != nil {
 		log.Fatalf("Unable to setup trace publishing: %v", err)
 	}
-	c, err := kncloudevents.NewDefaultClient()
+
+	handler := ochttp.Handler{
```

Comment:
you can use `CreateHandler()` from `eventing/pkg/kncloudevents/message_receiver.go` for creating the handler

Reply:
I'll take a look at that!  thanks for the pointer.